glean-core/rlb/src/private/ping.rs

```diff
@@ -42,7 +42,7 @@ impl PingType {
     /// # Returns
```

Comment:
As above

Reply:
though actually ... we should keep the return value here.
We should just strip it out in the functions above where we dispatch it.

reason: we need to know if something is successfully submitted so we can actually trigger an upload.

Reply:
We can still handle as we do for other bindings (within the upload mechanism / the glean singleton)

Reply:
Right, let's fix this up the way we need it when we need it.

Reply:
Cool, it will happen as part of https://bugzilla.mozilla.org/show_bug.cgi?id=1672958, which I'm taking next.